pyproject.toml

```diff
@@ -0,0 +1,50 @@
+[tool.poetry]
+name = "kintree"
+version = "0.3.6"
+description = "Fast part creation in KiCad and InvenTree"
+authors = ["Placeholder Name <placeholder@test.com>"]
+license = "GPL3"
+
+[tool.poetry.dependencies]
+
+python = ">=3.7,<3.10"
+altgraph = ">=0.17,<1.0"
+certauth = ">=1.3.0,<2.0"
+certifi = ">=2020.6.20,<2021.0"
+cffi = ">=1.14.1,<2.0"
+chardet = ">=3.0.4,<4.0"
+coverage = ">=5.2.1,<6.0"
```

Comment:
put this in the dev dependency.

Reply:
@eeintech Please review this and check which of these packages are actually used, either for dev / test purposes, or for actual implementation.

For example, I can't seem to find any use of the following packages:

- certifi
- cffi
- chardet
- coveralls
- inflection
- pycparser

Reply:
@randrej Apparently `certifi`, `cffi`, `chardet` and `inflection` are all dependencies of the Digi-Key API project. But it seems like `pycparser` serves no purpose so I removed it :+1:

Reply:
`coverage` can be installed when the tests are run only using the `inv test` command

Reply:
> @randrej Apparently `certifi`, `cffi`, `chardet` and `inflection` are all dependencies of the Digi-Key API project. But it seems like `pycparser` serves no purpose so I removed it +1

If that's the case, they can be removed from this package's dependency list. They will be installed as transitive dependencies anyway. How about the other ones? Are there any other packages that are dependencies of packages formerly included as submodules?

Reply:
Ok I did a big clean-up, you are right, no need of maintaining dependencies from other packages!